src/app/payment/paymentMethod/MolliePaymentMethod.tsx

```diff
@@ -66,21 +75,30 @@ const MolliePaymentMethod: FunctionComponent<MolliePaymentMethodsProps> = ({ ini
     }
 
     function isCreditCard(): boolean {
+
         return method.method === MolliePaymentMethodType.creditcard;
     }
 
+    function validateInstrument(_shouldShowNumber: boolean, selectedInstrument: CardInstrument) {
```

Comment:
the first param doesn't seem to be used? maybe remove?

Reply:
its used by Adyen, so thats why we can't remove it!